documentation/migration-guide-0.1.1-to-0.2.0.md

```diff
@@ -0,0 +1,30 @@
+# Migration guide from v0.1.1 to v0.2.0
+
+### Added seed job agent
+Now seed jobs are not built by master executors, but by dedicated agents deployed to Kubernetes. We disabled master executors for security reasons.
```

Comment:
`Now seed jobs are not built by master executors, but by dedicated agent deployed into Kubernetes. We disabled master executors for security reasons.`